rlinf/workers/actor/megatron_actor_worker.py

```diff
@@ -1162,38 +1162,44 @@ def run_inference(
             output_channel: The output channel to send results to.
             compute_ref_logprobs: Whether to compute reference logprobs.
         """
-        recv_batch_size = 0
-        while recv_batch_size < self.total_batch_size_per_dp:
-            batch, rollout_result = self.get_batch(input_channel)
-            recv_batch_size += rollout_result.num_sequence
+        with self.worker_timer():
+            recv_batch_size = 0
+            batches = []
+            rollout_results = []
+            while recv_batch_size < self.total_batch_size_per_dp:
```

Comment:
This effectively disables pipelining between inference and train. We need a parameter to control the batch granularity.